galaxy_ng/app/api/ui/views/settings.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+from rest_framework.permissions import IsAuthenticatedOrReadOnly
+from rest_framework.response import Response
+
+from galaxy_ng.app.api import base as api_base
+
+
+class SettingsView(api_base.APIView):
+    permission_classes = [IsAuthenticatedOrReadOnly]
+
+    def get(self, request, *args, **kwargs):
+        data = {}
+        keyset = [
+            "GALAXY_ENABLE_UNAUTHENTICATED_COLLECTION_ACCESS",
+            "GALAXY_ENABLE_UNAUTHENTICATED_COLLECTION_DOWNLOAD",
+            "GALAXY_FEATURE_FLAGS",
+        ]
+        data = {key: settings.get(key, None) for key in keyset}
```

Comment:
sorry I missed this in my previous comment, `as_dict()` so it is serializable to json
```suggestion
        data = {key: settings.as_dict().get(key, None) for key in keyset}
```